src/ServicePulse.Host/vue/src/components/monitoring/EndpointList.vue

```diff
@@ -30,23 +35,58 @@ onMounted(async () => {
     <!--Table headings-->
     <div class="table-head-row">
       <div class="table-first-col">
-        <endpoint-list-sortable-column>Endpoint name</endpoint-list-sortable-column>
+        <EndpointListSortableColumn :is-active="activeColumn === 'name'" :sort-by="'name'" @click="updateActiveColumn('name')">Endpoint name</EndpointListSortableColumn>
```

Comment:
If you drop the colon, you don't need the extra quotes:

```suggestion
        <EndpointListSortableColumn :is-active="activeColumn === 'name'" sort-by="name" @click="updateActiveColumn('name')">Endpoint name</EndpointListSortableColumn>
```